base-admin/linux-pam/spec

```diff
@@ -1,3 +1,4 @@
 VER=1.4.0
+REL=1
 SRCTBL="https://github.com/linux-pam/linux-pam/releases/download/v$VER/Linux-PAM-$VER.tar.xz"
```

Comment:
Update spec format please, and the same goes for the rest of the packages affected.